alwaysontop/main.js

```diff
@@ -0,0 +1,79 @@
+const electron = require('electron');
+const { BrowserWindow } = electron;
+
+const SIZE = {
+    width: 320,
+    height: 180
+};
+
+/**
+ * Handles new-window events for the main process in order to customize the
+ * BrowserWindow options of the always on top window. This handler will be
+ * executed in the context of the main process.
+ *
+ * @inheritdoc
+ */
+function onAlwaysOnTopWindow(
+        event,
+        url,
+        frameName,
+        disposition,
+        options) {
+    if (frameName === 'AlwaysOnTop') {
+        event.preventDefault();
+        event.newGuest = new BrowserWindow(
```

Comment:
When the event's default is prevented we are supposed to construct the BrowserWindow on our own and set it to the newGuest prop. Also this way we are able to set the BrowserWindow's options that we need (like always on top, frameless, etc).

Check https://github.com/electron/electron/blob/master/docs/api/window-open.md#using-chromes-windowopen-implementation and https://github.com/electron/electron/blob/master/docs/api/web-contents.md for more info.